docs/get-started.asciidoc

```diff
@@ -1,23 +1,172 @@
+====
+
+////
+TO DO:
+Anything else they need to do/have/know?
+For example, any dependencies (including specific supported Java versions)?
+////
+
+////
+✅ How to install the distro
+////
 [discrete]
 [[get-started-install]]
 === Install
 
-. Download the latest release https://mvnrepository.com/artifact/co.elastic.otel/elastic-otel-javaagent/latest[here].
-. Run the distro by using the `-javaagent:` JVM argument with the path to the distro jar:
+////
+✅ Step-by-step instructions
+////
+To get started with the distro:
+
+. Download the https://mvnrepository.com/artifact/co.elastic.otel/elastic-otel-javaagent/latest[latest release].
+. Run the distro using the `-javaagent:` JVM argument with the path to the distro jar:
 +
 [source,bash]
 ----
 java -javaagent:/path/to/agent.jar \
 -jar myapp.jar
 ----
 
+////
+✅ Any additional info related to installation
+////
+[NOTE]
+.The Elastic distro and the OpenTelemetry SDK
+====
+The distro includes a transitive dependency on the OpenTelemetry SDK, so you do not _need_ to add the
+OpenTelemetry SDK package to your project directly. However, you _can_ explicitly add the OpenTelemetry SDK as a dependency if you want to opt into newer SDK versions. If you do this, the OpenTelemetry SDK dependency must
+be defined _before_ the {distro_name} is defined.
+====
+
+////
+✅ Start-to-finish operation
+////
+[discrete]
+[[get-started-send-data]]
+=== Send data to Elastic
+
+After installing the distro, configure and initialize it to start
+sending data to Elastic.
+
+////
+TO DO:
+Are there any dependencies that need to be installed in addition to the distro?
+If yes, does it matter in what order dependencies are defined?
+If no, delete this section.
+////
+[discrete]
+[[get-started-add-dependencies]]
+==== Add dependencies
+
+////
+Step-by-step instructions
+////
+
+////
+TO DO:
+What does the _minimal_ configuration / setup look like?
+Are there any configuration options that are _required_?
+////
 [discrete]
 [[get-started-configure]]
-=== Configure
+==== Configure the distro
+
+////
+Step-by-step instructions
+////
+
+////
+TO DO:
+Does the user need to do something after configuring the distro
+for data to start being sent Elastic?
+If no, delete this section.
+////
+[discrete]
+[[get-started-initialization]]
+==== Initialize the distro
+
+////
+Step-by-step instructions
+////
+
+////
+Optional: Add an image
+////
+
+////
+TO DO:
+What should they expect to see after running this?
+How do they know it worked as expected?
+////
+[discrete]
+[[get-started-view-traces]]
+=== View traces
+
+To view the traces, you can use the Elastic APM UI:
+
+////
+Step-by-step instructions
+////
+
+////
+✅ What they should do next
+////
+[discrete]
+[[get-started-next-steps]]
+=== Next steps
+
+* <<configuration,Configure the distro>> to send data to Elastic.
+* Learn more about viewing and interpreting data in the {observability-guide}/apm.html[Observability guide].
```

Comment:
I wonder if we can link to a more actionable page (or pages). This link kinda drops the user at a useless landing page with generic info.

Reply:
I agree. I was thinking of linking to a page related to the APM UI, but we'll need to merge https://github.com/elastic/observability-docs/pull/3723 first.